packages/connector-saml/src/constant.ts

```diff
@@ -28,3 +27,128 @@ export const authnRequestBinding = ['HTTP-Redirect'] as const;
+    key: 'signInEndpoint',
+    required: true,
+  },
+  {
+    type: ConnectorConfigFormItemType.MultilineText,
+    label: 'x509Certificate',
+    key: 'x509Certificate',
+    required: true,
+    placeholder:
+      '-----BEGIN CERTIFICATE-----\nMIIDHTCCAgWg[...]jel7/YMPLKwg+Iau7\n-----END CERTIFICATE-----',
+  },
+  {
+    type: ConnectorConfigFormItemType.MultilineText,
+    label: 'idpMetadataXml',
+    key: 'idpMetadataXml',
+    required: true,
+  },
+  {
+    type: ConnectorConfigFormItemType.Text,
+    label: 'assertionConsumerServiceUrl',
+    key: 'assertionConsumerServiceUrl',
+    required: true,
+  },
+  {
+    type: ConnectorConfigFormItemType.Select,
+    label: 'requestSignatureAlgorithm',
+    key: 'requestSignatureAlgorithm',
+    selectItems: [
+      { value: 'http://www.w3.org/2000/09/xmldsig#rsa-sha1', title: 'RSA SHA1' },
+      {
+        value: 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256',
+        title: 'RSA SHA256',
+      },
+      {
+        value: 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha512',
+        title: 'RSA SHA512',
+      },
+    ],
+    defaultValue: 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256',
+  },
+  {
+    type: ConnectorConfigFormItemType.Select,
+    label: 'messageSigningOrder',
+    key: 'messageSigningOrder',
+    selectItems: [
+      { value: 'sign-then-encrypt', title: 'Sign then encrypt' },
+      {
+        value: 'encrypt-then-sign',
+        title: 'Encrypt then sign',
+      },
+    ],
+    defaultValue: 'sign-then-encrypt',
+  },
+  {
+    type: ConnectorConfigFormItemType.Switch,
+    label: 'signAuthnRequest',
+    key: 'signAuthnRequest',
+    defaultValue: false,
+  },
+  {
+    type: ConnectorConfigFormItemType.MultilineText,
+    label: 'privateKey',
+    key: 'privateKey',
+    required: true,
+    showConditions: [{ targetKey: 'signAuthnRequest', expectValue: true }],
+    placeholder:
+      '-----BEGIN RSA PRIVATE KEY-----\n[private-key-content]\n-----END RSA PRIVATE KEY-----',
+  },
+  {
+    type: ConnectorConfigFormItemType.Text,
+    label: 'privateKeyPass',
+    key: 'privateKeyPass',
+    showConditions: [{ targetKey: 'signAuthnRequest', expectValue: true }],
+  },
+  {
+    type: ConnectorConfigFormItemType.Switch,
+    label: 'encryptAssertion',
+    key: 'encryptAssertion',
+    defaultValue: false,
+  },
+  {
+    type: ConnectorConfigFormItemType.MultilineText,
+    label: 'encPrivateKey',
+    key: 'encPrivateKey',
+    required: true,
+    showConditions: [{ targetKey: 'encryptAssertion', expectValue: true }],
+    placeholder:
+      '-----BEGIN RSA PRIVATE KEY-----\n[private-key-content]\n-----END RSA PRIVATE KEY-----',
+  },
+  {
+    type: ConnectorConfigFormItemType.Text,
+    label: 'encPrivateKeyPass',
+    key: 'encPrivateKeyPass',
+    showConditions: [{ targetKey: 'encryptAssertion', expectValue: true }],
+  },
+  {
+    type: ConnectorConfigFormItemType.Text,
+    label: 'nameIDFormat',
+    key: 'nameIDFormat',
```

Comment:
Would you mind adding a TODO reminder for me since we have changed the type of nameIDFormat in [another PR](https://github.com/logto-io/connectors/pull/85)?